packages/storybook/stories/AdhocCalling.stories.mdx

```diff
@@ -12,37 +13,44 @@ import AdhocCallingSnippetText from '!!raw-loader!./snippets/AdhocCalling.snippe
-
 ## Overview
 
-A call can be started with a Teams user or a call queue by using their id. The id of the Teams user or call queue is
-added to the `CallParticipantsLocator` which is used as the locator in the
-[AzureCommunicationCallAdapterArgs](./?path=/docs/composite-adapters--page#azurecommunicationcalladapterargs)
-when creating your [CallAdapter](path=/docs/composite-adapters--page#call-adapter).
-In the current beta version, only one Teams user can be called. A Teams call queue can also be called. For more
-information on call queues, refer to this
-[Learn documentation](https://learn.microsoft.com/en-us/microsoftteams/create-a-phone-system-call-queue)
+A call can be started with a Teams user or a call queue by using their id. You can call a teams user or a Teams voice application
+like a call queue or auto attendant. These calls it is important to note can only be one Teams voice application, or, multiple teams users. You cannot call multiple
+Teams voice applications at the same time.
+
+## Calling a Teams voice application
+
+To call a Teams voice application like a call queue or auto attendant, you need to use the id of the Teams resource account. Check out this [tutorial](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/get-started-teams-auto-attendant#find-object-id-for-auto-attendant) on how to find
+this id. The id of the Teams resource account should be stored in the type of `MicrosoftTeamsAppIdentifier[]` and given to the `targetCallees` property
+of the [CallAdapter](?path=/docs/composite-adapters--page#call-adapter) when creating a call. Now when the [CallComposite](?path=/docs/composites-call-basicexample--basic-example) starts up it will call the Teams voice application
+from the configuration screen. See the following snippet for an example.
 
-<MessageBar messageBarType={MessageBarType.warning}>
-  Note: Teams user ids need to be in format '8:orgid:UUID' when used in the CallParticipantsLocator. For example,
-  '8:orgid:87d349ed-44d7-43e1-9a83-5f2406dee5bd'.
-</MessageBar>
-<br />
 <MessageBar messageBarType={MessageBarType.warning}>
   Note: Teams resource account ids linked to a call queue need to be in format '28:orgid:UUID' when used in the
   CallParticipantsLocator. For example, '28:orgid:87d349ed-44d7-43e1-9a83-5f2406dee5bd'.
 </MessageBar>
 
-## Sample code
-
-The code snippet below is an example of setting up your CallComposite for adhoc calling.
-
-<Source code={AdhocCallingSnippetText} />
+<Source code={AdhocCallingVoiceAppSnippetText} />
 
 The two callback functions `onFetchProfile` and `onFetchAvatarPersonaData` provided at the bottom of the snippet are
 for customizing display names and avatars of users, respectively.
 
+## Calling a Teams user
+
+<SingleLineBetaBanner />
+
+Like a Teams voice application you can call a Teams user by using their id. The id of the Teams user should be stored in the type of `MicrosoftTeamsUserIdentifier[]`.
```

Comment:
missing comma. 'Like a Teams voice application, you can call a Teams user by using their id'